project/CopyrightHeader.scala

```diff
@@ -1,9 +1,3 @@
-/*
- * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
- */
-
-package akka.grpc
-
 import sbt._, Keys._
```

Comment:
Hmm, do we really want to remove the header? Shouldn't we have one dated correctly, though?

Reply:
We aren't too good att headers in build files. But I added it back.